articleEditor/editor.js

```diff
@@ -0,0 +1,346 @@
+        SearchArticles();
+        CheckIfHiddenCardsContent();
+        reattachListeners();
+        return true;
+    }
+    return false;
+}
+function updateDate(date) {
+
+    let now = new Date();
+
+    let diffMillis = now - date;
+    if (diffMillis < 1000) return "right now";
+    if (diffMillis < 1000 * 60) return `${Math.round((diffMillis / 1000))} sec. ago`;
+    if (diffMillis < 1000 * 60 * 60) return `${Math.round(diffMillis / 1000 / 60)} min. ago`;
+    if (diffMillis < 1000 * 60 * 60 * 24) return `${Math.round(diffMillis / 1000 / 60 / 60)} hour ago`;
+    const twoD = (number) => number.toString().slice(-2).padStart(2, '0');
+    date = new Date(date);
+    return twoD(date.getDate()) + '.' +
+        twoD(date.getMonth() + 1) + '.' +
+        twoD(date.getFullYear());
+}
+function warning(message) {
+
+    document.getElementsByClassName(`warning`)[0].innerHTML = message;
+}
+function counter() {
+
+    let count = localStorage.getItem(`articlesCounter`) ?
+        localStorage.getItem(`articlesCounter`) : 0;
+    return () => count++;
+}
+function addArticleToStorage(article) {
+
+    window.articlesFromLS.push(article);
+    localStorage.setItem(`Articles`, JSON.stringify(window.articlesFromLS));
+    console.log(window.articlesFromLS);
+}
+
+function ShowArticles(arrayArticle) {
+
+    document.querySelector(`.cardsContainer`).innerHTML = "";
+
+    for (let i = 0; i < arrayArticle.length; i++) {
+        createCard(arrayArticle[i]);
+        document.querySelectorAll(`.card-text`)[i].setAttribute(`data-article-number`, i);
+    }
+}
+function createCard(values) {
+
+    document.querySelector(`.cardsContainer`).appendChild(Card(values));
+}
+function Card(values) {
+
+    let div = document.createElement('div');
+
+    div.className = "card col-xs-12 col-sm-6 col-md-4 col-lg-4 col-xl-3 mb-4";
+    div.innerHTML = `<div class="card-body h-100 form-inline">
+            <a class="w-100"> <span class="card-title w-75 h5 float-left">${values.articleName}</span>
+                <span class="badge badge-secondary w-25 float-right">${values.articlenickName}</span>
+            </a>
+            <p class="card-text w-100">${values.articleDescription}</p>
+        </div>
+        <div class="card-footer  form-inline p-2 rounded">
+            <small class="text-muted w-50 mr-auto">Last updated ${updateDate(values.articleLastDate)}</small>
+            ${(values.articlenickName === window.nickName) ? `
+            <div data-fix-id="${values.articleId}" class="w-50  form-inline">
+                 <div  data-fix="edit" class="m-auto div_img">
+                    <img src="../src/edit_pen.png" alt="Редактировать"  class="img">
+                 </div>
+                 <div  data-fix="delete" class="m-auto div_img">
+                     <img src="../src/delete.png" alt="Удалить"  class="img">
+                 </div>
+            </div>` : `<div class="m-auto x"></div>`}
+        </div>`;
```

Comment:
Do not use html inside js, it is a bad practice that triggers many impacts.
I propose to create hidden html card, replace needed variables by a placeholders.
Then, just clone this card and replace all placeholder by your compiled variables, remove hidden class and place as new card.